0-bootstrap/main.tf

```diff
@@ -60,7 +60,6 @@ module "seed_bootstrap" {
   project_prefix                 = var.project_prefix
   encrypt_gcs_bucket_tfstate     = true
   key_rotation_period            = "7776000s"
-  kms_prevent_destroy            = !var.bucket_tfstate_kms_force_destroy
```

Comment:
This three inputs work as a set: 

```hcl
module "kms" {
  count   = var.encrypt_gcs_bucket_tfstate ? 1 : 0 <======
  source  = "terraform-google-modules/kms/google"
  version = "~> 2.1"

  project_id           = module.seed_project.project_id
...
  key_rotation_period  = var.key_rotation_period <======
...
  encrypters = [
    "serviceAccount:${data.google_storage_project_service_account.gcs_account.email_address}",
  ]
  prevent_destroy = var.kms_prevent_destroy <======
}
```
https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/main.tf#L110C1-L115C56

Removing only the `kms_prevent_destroy` will make the build fail on the destroy of  `0-bootstrap`

Reply:
I undid this change with commit 29e0ec8